views/problem_set.ejs

```diff
@@ -36,19 +36,11 @@
                         <th>Difficulty</th>
                     </tr>
                 </thead>
-                <tbody>
-                    <tr>
-                        <td><a href="/problem/4">4</a></td> 
-                        <td><a href="/problem/4">Dummy Name</a></td>
-                        <td>math, brute force</td>
-                        <td>easy</td>
-                    </tr>
-                    <tr>
-                        <td><a href="/problem/6">6</a></td> 
-                        <td><a href="/problem/6">Again Dummy Name</a></td>
-                        <td>math, brute force</td>
-                        <td>easy</td>
-                    </tr>
+                <tbody id="table_js">
+                    <script>
+                        var table = "{{table}}";
+                        console.log(table);
+                    </script>
                 </tbody>
```

Comment:
Also, there is no need for <script> as it is not html but ejs. You can directly access the variables in `<%= ... >`